packages/ra-ui-materialui/src/button/SaveButton.tsx

```diff
@@ -112,24 +105,11 @@ export const SaveButton = <RecordType extends RaRecord = any>(
             aria-label={displayedLabel}
             disabled={disabled}
             onClick={handleClick}
+            label={displayedLabel}
             // TODO: find a way to display the loading state (LoadingButton from mui Lab?)
             {...sanitizeButtonRestProps(rest)}
         >
-            {saving ? (
-                <CircularProgress
-                    size={18}
-                    thickness={2}
-                    className={SaveButtonClasses.leftIcon}
-                />
-            ) : (
-                cloneElement(icon, {
-                    className: classnames(
-                        SaveButtonClasses.leftIcon,
-                        SaveButtonClasses.icon
-                    ),
-                })
-            )}
-            {displayedLabel}
+            {saving ? <CircularProgress size={18} thickness={2} /> : icon}
```

Comment:
Aren't we missing `className={SaveButtonClasses.leftIcon}` here?

Reply:
Good catch

Reply:
Actually not an issue as the margin is applied on the button label